packages/backend/src/asset/service.test.ts

```diff
@@ -213,15 +243,22 @@ describe('Asset Service', (): void => {
     getPageTests({
       createModel: () => createAsset(deps),
       getPage: (pagination?: Pagination, sortOrder?: SortOrder) =>
-        assetService.getPage(pagination, sortOrder)
+        assetService.getPage({
```

Comment:
Should we test for different behavior between providing a tenantId versus not providing one?